src/bin/solang.rs

```diff
@@ -177,6 +177,12 @@ fn main() {
                             .short('g')
                             .long("generate-debug-info")
                             .hide(true),
+                    )
+                    .arg(
+                        Arg::new("LOGAPIRETURNS")
+                            .help("Log the return codes of runtime API calls in the environment")
+                            .long("log-api-calls")
```

Comment:
How about `--log-api-returns` or `--log-api-return-codes` ?

I imagine at some point we may want another option which logs the api calls with their arguments, which would be called `--log-api-calls`, I suppose.

Reply:
ACK